pom.xml

```diff
@@ -3,7 +3,7 @@
 
   <groupId>com.plaid</groupId>
   <artifactId>plaid-java</artifactId>
-  <version>5.1.6-SNAPSHOT</version>
+  <version>5.2.1-SNAPSHOT</version>
```

Comment:
idk much about java but is this supposed to be 5.2.0 instead of 5.2.1?

Reply:
This is the default variable for what the next version should be, so 5.2.1 is correct! hahaha nice catch though